lib/decidim/cdtb/upgrades/validate_migrations_task.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Cdtb
+    module Upgrades
+      # Validates that all Decidim modules have the migrations installed.
+      #
+      class ValidateMigrationsTask < ::Decidim::Cdtb::Task
+        def initialize
+          progress_bar= { title: "Modules" }
+          super("VALIDATE MODULES MIGRATIONS", progress_bar: progress_bar)
+        end
+
+        def prepare_execution(_ctx)
+          log_task_info("Searching gems...")
+
+          all_railties= Rails.application.migration_railties
+          railties_w_migrations= all_railties.select do |railtie|
+            railtie.respond_to?(:paths) && railtie.paths["db/migrate"].first.present?
+          end
+          @gem_names= railties_w_migrations.map(&:railtie_name)
+
+          log_task_info("Found #{@gem_names.size} gems with migrations...")
+        end
+
+        def total_items
+          2
```

Comment:
What is that?

Reply:
A magic number... I've created a constant thanks!